packages/mgt-components/src/components/mgt-teams-channel-picker/mgt-teams-channel-picker.ts

```diff
@@ -757,6 +757,16 @@ export class MgtTeamsChannelPicker extends MgtTemplatedComponent {
     this.resetFocusState();
   }
 
+  /**
+   * Clears the selectedItem state.
+   *
+   * @memberof MgtTeamsChannelPicker
+   */
+  clearSelectedItem() {
+    this.removeSelectedChannel(undefined);
+    this.fireCustomEvent('selectionChanged', this.selectedItem);
```

Comment:
My read is that this event is already fired when `selectChannel` is called by `removeSelectedChannel`, so this line should probably be removed.